front_end/src/app/admin/rooms/room-group-list.component.html

```diff
@@ -0,0 +1,114 @@
+<div class="wrapper rooms">
+  <nav class="navbar navbar-light bg-faded">
+    <a class="navbar-brand mr-2 hidden-sm-down" [routerLink]="['/admin']">
+      <svg class="admin-logo" viewBox="0 0 45 45" xmlns="http://www.w3.org/2000/svg">
+        <g fill="none" fill-rule="evenodd">
+          <path d="M22.414 0C10.037 0 0 10.034 0 22.42 0 34.8 10.037 44.843 22.414 44.843c12.38 0 22.425-10.042 22.425-22.425C44.84 10.033 34.792 0 22.412 0zm-.08 27.03l-7.856 7.863-4.69-4.702 7.85-7.853L9.91 14.61l4.695-4.707 7.73 7.734 7.85-7.852 4.7 4.7-7.844 7.85 8.015 8.023-4.702 4.702-8.018-8.03z" fill="#B05C30"/>
+          <path fill="#FFF" d="M22.334 27.03l-7.856 7.863-4.69-4.702 7.85-7.853L9.91 14.61l4.695-4.707 7.73 7.734 7.85-7.852 4.7 4.7-7.844 7.85 8.015 8.023-4.702 4.702"/>
+        </g>
+      </svg>
+      <span>eCheck</span>
+    </a>
+    <ul class="nav navbar-nav">
```

Comment:
@timmyg Is this used anywhere else if so it should probably be its own component.